include/ddc/periodic_sampling.hpp

```diff
@@ -127,8 +131,11 @@ class PeriodicSampling : detail::PeriodicSamplingBase
         {
             return m_origin
                    + continuous_element_type(
-                             static_cast<int>((icoord.uid() + m_n_period / 2 - 1) % m_n_period)
-                             - static_cast<int>(m_n_period / 2 - 1))
+                             static_cast<int>(
+                                     (icoord.uid()
+                                      + (m_n_period - static_cast<int>(m_odd_period)) / 2)
+                                     % m_n_period)
+                             - static_cast<int>((m_n_period - static_cast<int>(m_odd_period)) / 2))
```

Comment:
This code was not aligned with `numpy.fftfreq` convention to describe the periodic window. Now it is

Reply:
https://numpy.org/doc/stable/reference/generated/numpy.fft.fftfreq.html

Reply:
I guess it is necessary to get proper convergence for spectral methods ?

Anyway it is a good thing

Reply:
That's a very good question, I am not sure of the answer. The computation is numerically different when we divide by /k^2 for sure, but I wonder if by the magic of Fourier transform it could become the same after performing the iFFT. In my understanding the choice of position of the periodic window is supposed to be arbitrary, so it would be good if it has no mathematical impact. But I have no proof for that :/